keywords/MobileRestClient.py

```diff
@@ -1020,7 +1037,7 @@ def purge_docs(self, url, db, docs):
 
         return purged_docs
 
-    def update_docs(self, url, db, docs, number_updates, delay=None, auth=None, channels=None):
+    def update_docs(self, url, db, docs, number_updates, delay=None, auth=None, channels=None, property_updater=None):
```

Comment:
Any property can be updated by passing a function. See https://github.com/couchbaselabs/mobile-testkit/blob/feature/sg-sdk-tests/testsuites/syncgateway/functional/tests/test_xattrs.py#L1109-L1112 for a sample usage